src/parsers/pm_io/psse.jl

```diff
@@ -315,17 +319,17 @@ function _psse2pm_load!(pm_data::Dict, pti_data::Dict, import_all::Bool)
                 if (load["IP"] != 0.0) || (load["IQ"] != 0.0)
                     # Uses matpower transformation instead of pd = real(V*I) and qd = imag(V*I)
                     # where I and V are in vector form.
-                    sub_data["pd"] += bus_vm * load["IP"]
-                    sub_data["qd"] += bus_vm * load["IQ"]
-                    @warn "Load id = $(sub_data["index"]) detected as I Load  IP = $(load["IP"]) IQ = $(load["IQ"]). Converting to Power Load Pd = $(bus_vm*load["IP"]) Qd = $(bus_vm*load["IQ"]) using Vm = $(bus_vm)"
+                    sub_data["pd"] += load["IP"]
```

Comment:
@scdhulipala can you undo this change to merge the other fixes?

Reply:
@jd-lara  I have to make a few changes to HVDC parsing as well. I will make those changes and undo these so we can merge. I can get to this next week because I'm trying to wrap up some PR100 tasks this week.